generator/dart/generated/google_cloud_functions_v2/lib/cloudfunctions.dart

```diff
@@ -327,33 +327,33 @@ class Function$ extends Message {
 }
 
 /// Describes the current state of the function.
-class Function$$State extends Enum {
+class Function$State extends Enum {
   /// Not specified. Invalid state.
-  static const stateUnspecified = Function$$State('STATE_UNSPECIFIED');
+  static const stateUnspecified = Function$State('STATE_UNSPECIFIED');
 
   /// Function has been successfully deployed and is serving.
-  static const active = Function$$State('ACTIVE');
+  static const active = Function$State('ACTIVE');
 
   /// Function deployment failed and the function is not serving.
-  static const failed = Function$$State('FAILED');
+  static const failed = Function$State('FAILED');
 
   /// Function is being created or updated.
-  static const deploying = Function$$State('DEPLOYING');
+  static const deploying = Function$State('DEPLOYING');
 
   /// Function is being deleted.
-  static const deleting = Function$$State('DELETING');
+  static const deleting = Function$State('DELETING');
 
   /// Function deployment failed and the function serving state is undefined.
   /// The function should be updated or deleted to move it out of this state.
-  static const unknown = Function$$State('UNKNOWN');
+  static const unknown = Function$State('UNKNOWN');
 
-  const Function$$State(super.value);
+  const Function$State(super.value);
 
-  factory Function$$State.fromJson(String json) => Function$$State(json);
+  factory Function$State.fromJson(String json) => Function$State(json);
```

Comment:
(Could reuse existing instances, instead of creating a new one with the same value.)